build/version.go

```diff
@@ -55,7 +55,7 @@ func (ve Version) EqMajorMinor(v2 Version) bool {
 // APIVersion is a semver version of the rpc api exposed
 var APIVersion Version = newVer(0, 3, 0)
 
-//nolint:varcheck
+//nolint:varcheck,deadcode
```

Comment:
if this is dead code shouldnt we just remove it?

Reply:
It is about maskMajor and maskMinor (or two other), and I feel like it is fine to have them for the sake of completeness.